stetho/src/main/java/com/facebook/stetho/common/ListAccumulator.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.common;
+
+import java.util.List;
+
+public interface ListAccumulator<E> extends List<E>, Accumulator<E> {
```

Comment:
...why not just go straight after the `ArrayListAccumulator`?  This diff appears to be way overrotating on generalization that isn't even used.  This generalization has a real cost to our released jar size which folks are sensitive to.